changes/443.feature

```diff
@@ -0,0 +1 @@
+Make a error message when raise IntegrityError due to missing scaling groups.
```

Comment:
Since this is a Markdwon-formatted text line, let's refine it with more details:
```suggestion
Make an explicit error message upon `IntegrityError` due to missing scaling groups when handling agent heartbeats.
```